src/main/webapp/app/model/iRContext.js

```diff
@@ -303,6 +305,11 @@ cap.model("IRContext", function ($q, $filter, $interval, $location, $routeParams
         }
       });
 
+      fixityPromise.then(function(apiRes) {
+        var newContext = angular.fromJson(apiRes.body).payload.IRContext;
```

Comment:
Need to check if the status was SUCCESS here.

Reply:
This is sufficient, as it restores the previous functionality.

Reply:
Is error prone is sufficient, sure.

Reply:
It is.